mdx-models/src/main/java/com/mx/path/model/mdx/model/profile/Email.java

```diff
@@ -51,6 +51,14 @@ public void setId(String id) {
     this.id = id;
   }
 
+  public String getRank() {
```

Comment:
If this has a set list of `rank` why not use an enumerator? 
```
| Type      | Description             |
| --------- | ----------------------- |
| PRIMARY   | Primary Email address   |
| SECONDARY | Secondary Email address |
| TERTIARY  | Tertiary Email address  |
```

Reply:
Similar to EmailType below

Reply:
Hmm, EmailType doesn't seem to be used.

Reply:
We can, but this was not added as enum when it was added to Phone number model. https://github.com/mxenabled/path-mdx-model/blob/4f6511e5dcb22762fb7a6fe25284346bb56d9be5/mdx-models/src/main/java/com/mx/path/model/mdx/model/profile/Phone.java#L16

So I was trying to keep it consistent.

Reply:
I am not sure about how it turned out to be a string and not an enum, but the story description originally had it as an enum https://mxcom.atlassian.net/browse/HW2-296. and it was implemented like that in HW accessor.
@RyanJones08 / @gerfboy do we want to proceed or switch to enum?

Reply:
Ryan and I think it's best to change these to enum now because we probably won't remember to come back and fix it later.

Reply:
Updated.